source/integrations-guide/integrations-guide-index.rst

```diff
@@ -6,370 +6,127 @@ Integrations Guide
-
-Learn more about `building your own plugin <https://developers.mattermost.com/integrate/plugins/>`_.
+Building a custom plugin is a **software development** task, using ``Go`` for the server-side functionality and optionally ``TypeScript/React`` for UI components. 
 
 Webhooks
 ~~~~~~~~
 
-Webhooks enable Mattermost to send and receive real-time data between your workspace and external applications. Webhooks require minimal coding and are easy to set up with virtually any tool or platform because they use lightweight HTTP POST requests with JSON payloads.
-
-Additionally, Mattermost webhook payloads are :ref:`fully compatible <integrations-guide/incoming-webhooks:slack compatibility>` with Slack’s webhook format to make migration easier.
+Learn more about :doc:`Mattermost webhooks </integrations-guide/webhook-integrations>`.
 
 Incoming Webhooks
-^^^^^^^^^^^^^^^^^
-
-**Technical complexity: No-code**
-
-Incoming webhooks allow external applications to post messages into Mattermost channels and direct messages. They are a simple way to receive notifications and data from other services in real-time.
+^^^^^^^^^^^^^^^^^^
 
-Using incoming webhooks in Mattermost requires only basic setup. You generate a webhook URL using the Mattermost interface, then point another service to send data to that address. No coding is required if your external service triggering the events is able to send data via webhooks or HTTP POST requests, which most modern applications and platforms support. Setting this up usually involves pasting the Mattermost webhook URL into the service’s settings and selecting what type of events you want it to send. 
+**Technical complexity:** :ref:`No-code <no-code>`
 
-Learn more about :doc:`incoming webhooks </integrations-guide/incoming-webhooks>`.
-
-**Examples**
-
-Here are some example use cases for incoming webhooks in Mattermost:
-
-- **Monitoring alerts**  
-   Send real-time alerts from monitoring systems (such as Prometheus or Datadog) into a dedicated Mattermost channel so your team is immediately notified about system issues or downtime.
-
-- **Build and deployment notifications**  
-   Post automated updates from CI/CD pipelines (such as Jenkins or GitLab CI) to a channel, keeping developers informed of build status, test results, and deployment progress.
-
-- **Customer support updates**  
-   Forward new support ticket notifications from systems like Zendesk or ServiceNow into a support channel, ensuring the team can respond quickly to incoming requests.
+:doc:`Incoming webhooks </integrations-guide/incoming-webhooks>` allow external applications to post messages into Mattermost channels and direct messages. They are a simple way to receive notifications and data from other services in real-time and require only basic setup.
 
 Outgoing Webhooks
```

Comment:
For this brochure version the outgoing webhooks section feels a bit verbose and long. Perhaps we move this content back to the dedicated doc and make this more summarized